src/qt/signverifymessagedialog.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "base58.h"
 #include "init.h"
+#include "main.h" // For strMessageMagic
```

Comment:
That's sad, we should move that elsewhere (not a criticism on your PR, just noticing).